### ISSUE TITLE
pyplot.errorbar attempts to plot 0 on a log axis in SVGs

### ISSUE THREAD

**Issue by BrianLandry** [matplotlib/matplotlib#6563]
pyplot.errorbar will attempt to plot a 0 point errorbar on a log axis when saving as a SVG, despite 0 not existing on a log axis. Although the SVG will be visualized correctly by chrome ect., it will crash inkscape, and be unable to be moved in illustrator. This can be accomplished with the following code

````
import matplotlib.pyplot as plt
plt.errorbar([0,1],[2,3],([0.5, 0.5],[0.5, 0.5]))
plt.xscale('log')
plt.savefig('example.svg')
````

This will generate a SVG with the following line:

````
<path clip-path="url(#p8e1df78064)" d="M -116021015.957275 252 
" style="fill:none;stroke:#0000ff;"/>
````

Where I think the " -116021015.957275" value is an attempt to plot at the 0 point on the log axis, which does not exist. Removing the zero point alleviates the reported problems

This was done in Windows 10, Python 2.7.11, Matplotlib 1.5.1, installed via Anaconda 4.0.0

Edit: This also appears to be an issue with pyplot.plot. However, for some pyplot.plot plots I do not get errors when working with some of them in illustrator, but regardless, they all still contain the highly negative location value in their XML.

**Comment by tacaswell** [comment]
Do pngs render correctly?

**Comment by BrianLandry** [comment]
Yes. SVGs even render correctly when viewed in chrome or IPython notebook.
The bug only has consequence when vector editors try to work the the SVGs

**Comment by tacaswell** [comment]
Can you try the 2.0.0b1 or 1.5.1rc2 release  (`conda install matplotlib -c conda-forge -c conda-forge/label/rc`)?  There was some work done on clipping with large numbers + vector backends (driven by pdf issues, but https://github.com/matplotlib/matplotlib/pull/5911).

**Comment by BrianLandry** [comment]
I used your command to install maptlotlib 2.0.0b1. This appeared to address this issue. I could no longer find the extreme value in the SVG xml and the svg functioned correctly when loaded into illustrator suggesting that the problem was fixed.

Thanks for the support! Feel free to close the issue.

**Comment by tacaswell** [comment]
Closed by #5911 .

**Comment by tacaswell** [comment]
@BrianLandry Thank you for reporting this bug and following up to check if it was already fixed.  We really do greatly appreciate both of those things!
